ontopy/utils.py

```diff
@@ -835,3 +835,34 @@ def recur(o):
                     )
 
     return layout
+
+
+def copy_annotation(onto, src, dst):
+    """In all classes and properties in `onto`, copy annotation `src` to `dst`.
+
+    The `src` and `dst` can either be provided as a label string or a full IRI.
+    """
+    if onto.world[src]:
+        src = onto.world[src]
+    else:
+        src = onto[src]
+
+    if onto.world[dst]:
+        dst = onto.world[dst]
+    elif dst in onto:
+        dst = onto[dst]
+    else:
+        if "://" not in dst:
+            raise ValueError(
```

Comment:
I think we should have tests for errors that we expect may happen.

Reply:
Isn't this such a test? 

What other errors would you expect?